README.md

```diff
@@ -0,0 +1,58 @@
+# Tic-tac-toe
```

Comment:
READMEs should provide information on technology (e.g. which testing framework did you use?) and helpful commands (e.g. how do you run tests?).

Reply:
done!